nest-city-account/src/utils/subservices/line-logger.subservice.ts

```diff
@@ -1,21 +1,22 @@
-import { Injectable, LoggerService } from '@nestjs/common'
 import * as process from 'node:process'
+
+import { LoggerService } from '@nestjs/common'
+
 import { escapeForLogfmt, isLogfmt, ToLogfmt } from '../logging'
 
 function getCurrentDateTime(): string {
   const now = new Date()
-  return now.toLocaleString('en-US', {
+  return now.toLocaleString('en-GB', {
```

Comment:
Can we stick to something without ambiguous slashes in the date (as current implementation returns `26/05/2025, 14:30:45`), preferably ISO 8601? Maybe even like this 
`return now.toISOString(); // e.g., "2025-05-26T14:30:45.123Z")`
or any other reasonable 24h time format alongside this `YYYY-MM-DD` date, may even be a local time, if there is no need for Z time.